hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMDirectoriesPurgeRequestWithFSO.java

```diff
@@ -148,7 +151,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
         getOmRequest());
     OMClientResponse omClientResponse = new OMDirectoriesPurgeResponseWithFSO(
         omResponse.build(), purgeRequests, ozoneManager.isRatisEnabled(),
-            getBucketLayout(), volBucketInfoMap, omFromSnapshot);
+            getBucketLayout(), volBucketInfoMap, rcOmFromSnapshot);
```

Comment:
In OMKeyPurgeRequest.java, you changed the parameter from snapshot to snapshotInfo.  Do we want to do the same thing here?

Reply:
Yep you are right I will need to do that.